dependencies.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/bitrise-io/go-utils/v2/command"
+)
+
+// EnsureDependenciesOpts ...
+type EnsureDependenciesOpts struct {
+	GemVersions gemVersions
+	UseBundler  bool
+}
+
+// InstallDependencies ...
+func (f FastlaneRunner) InstallDependencies(config Config, opts EnsureDependenciesOpts) error {
+	// Install desired Fastlane version
+	if opts.UseBundler {
+		f.logger.Infof("Install bundler")
+
+		// install bundler with `gem install bundler [-v version]`
+		// in some configurations, the command "bundler _1.2.3_" can return 'Command not found', installing bundler solves this
+		cmds := f.rbyFactory.CreateGemInstall("bundler", opts.GemVersions.bundler.Version, false, true, &command.Opts{
+			Stdout: os.Stdout,
+			Stderr: os.Stderr,
+			Stdin:  nil,
```

Comment:
No need to specify the options with default values (`Stdin` and `Env`).
Also pls, update the below command errors too.